docs/features/optin.md

```diff
@@ -0,0 +1,104 @@
+# Opt in to Enable Experimental Direct Controllers
+
+> `Feature State`: `alpha` as of version v1.123.1
+
+KCC can be configured to enable experimental versions of direct controllers for
```

Comment:
```suggestion
Config Connector can be configured to enable experimental versions of direct controllers for
```

Reply:
Curious, why is this change suggestion necessary?

Reply:
I just remembered -- this is our guidance for product naming (as mentioned by @maqiuyujoyce in a previous PR) https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2393#discussion_r1704740437

Reply:
Fixed